mayastor-test/e2e/install/deploy/pool.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "openebs.io/v1alpha1"
+kind: MayastorPool
+metadata:
+  name: pool-${NODE_NAME}
+  #generateName: pool-${NODE_NAME}
+  namespace: mayastor
+spec:
+  node: ${NODE_NAME}
+  disks: ["/dev/sda"]
```

Comment:
could this also be an env variable which we specify when running the tests?

Reply:
Deferring,  because it may be better enumerate available storage devices on the nodes and create a pool for each device found as part of the install test.